projects/bird-atlas/src/app/+species/species-info/species-info.component.html

```diff
@@ -0,0 +1,7 @@
+<ng-container *ngIf="taxon$ | async; let taxon">
+  <h1>{{ getVernacularName(taxon) }} ({{ taxon.scientificName }})</h1>
+  <h3>{{ getForeignVernacularNames(taxon) }}</h3>
```

Comment:
Usually not good to have functions in templates but doesn't seem like it causes performance problems here

Reply:
Yea it's not a problem imo since it only gets run once and the alternative is a bit uglier code wise imo. In general should aim to use OnPush everywhere and then functions in templates are not really a problem except in rare circumstances.